operators.go

```diff
@@ -302,6 +310,11 @@ type (
 		col    string
 		values []interface{}
 	}
+
+	like struct {
```

Comment:
there's no need for a new struct, actually, there is `colOp` for that https://github.com/src-d/go-kallax/blob/master/operators.go#L286